randovania/games/samus_returns/pickup_database/pickup-database.json

```diff
@@ -160,9 +162,10 @@
             "progression": [
                 "Charge"
             ],
-            "default_shuffled_count": 0,
-            "default_starting_count": 1,
-            "preferred_location_category": "major"
+            "default_shuffled_count": 1,
```

Comment:
Out of curiosity, what's the reasoning for not starting with it anymore? at least until there is a patch that changes most of the charge beam doors.

Reply:
I accidentally set it as a starting item, which it's not. Just correcting a previous pr's mistake.